partiql-lang/src/main/kotlin/org/partiql/lang/errors/ErrorCode.kt

```diff
@@ -1218,6 +1218,13 @@ enum class ErrorCode(
         ErrorBehaviorInPermissiveMode.RETURN_MISSING
     ),
 
+    EVALUATOR_NON_SINGLETON_COLLECTION(
```

Comment:
`EVALUATOR_COLLECTION_COERCION_TO_SCALAR_ERROR`?

There seems to be references to both _scalar_ and _singleton_; can we stick to one. I think _scalar_ is what spec. refers to.